parsers/src/main/java/vct/parsers/CMLtoCOL.scala

```diff
@@ -1061,8 +1061,6 @@ class CMLtoCOL(fileName: String, tokens: CommonTokenStream, parser: CParser)
       create expression(Length, expr(exp))
     case ValPrimary14("\\old", "(", exp, ")") =>
       create expression(Old, expr(exp))
-    case ValPrimary15("\\id", "(", exp, ")") =>
-      create expression(Identity, expr(exp))
```

Comment:
Can you also remove it from the specification grammar?

Reply:
@raulmonti I removed it from the specification grammar as requested. It should be ok to merge now (I think).

Reply:
Something broke :S no tests are passing @Drevanoorschot  and @niomaster